gobblin-runtime/src/main/java/gobblin/runtime/JobListener.java

```diff
@@ -19,7 +19,14 @@
   /**
    * Called when a job is completed.
    *
-   * @param jobState Job state
+   * @param jobState a {@link JobState} object
    */
-  public void jobCompleted(JobState jobState);
+  public void onJobCompletion(JobState jobState);
```

Comment:
I would rather keep the new name since currently this interface and all its implementation classes are only used internally, so it's probably the best time to correct the name (`onJobCompletion` sounds more appropriate for the name of a callback method) before it's used publicly.